build.gradle.kts

```diff
@@ -52,6 +54,13 @@ application {
 
 kotlin {
     jvmToolchain(21)
```

Comment:
What about just lowering the toolchain version here?

Reply:
I'd like to stay up-to-date with the tools/frameworks we use as much as possible, so I think lowering the target JRE version, rather than downgrading the JDK we use for compiling is preferable.